mini_messenger/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+
+class CustomUser(User):
+    contacts = models.ManyToManyField(to='CustomUser')
+
+
+class Dialog(models.Model):
+    users = models.ManyToManyField(to=CustomUser)
+
+
+class Message(models.Model):
+    text = models.TextField()
+    datetime_sent = models.DateTimeField(timezone.now)
```

Comment:
created_at

Reply:
models.DateTimeField(timezone.now) what is mean? is auto_now or auto_now_add. Need more concrete